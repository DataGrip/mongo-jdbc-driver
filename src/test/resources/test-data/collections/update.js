// before
db.books.insertMany([
    {
        "_id" : 1,
        "item" : "TBD",
        "stock" : 0,
        "info" : { "publisher" : "1111", "pages" : 430 },
        "tags" : [ "technology", "computer" ],
        "ratings" : [ { "by" : "ijk", "rating" : 4 }, { "by" : "lmn", "rating" : 5 } ],
        "reorder" : false
    },
    {
        "_id" : 2,
        "item" : "XYZ123",
        "stock" : 15,
        "info" : { "publisher" : "5555", "pages" : 150 },
        "tags" : [ ],
        "ratings" : [ { "by" : "xyz", "rating" : 5 } ],
        "reorder" : false
    }
]);
// command
db.books.update(
    { _id: 1 },
    {
        $inc: { stock: 5 },
        $set: {
            item: "ABC123",
            "info.publisher": "2222",
            tags: [ "software" ],
            "ratings.1": { by: "xyz", rating: 3 }
        }
    }
);
// clear
db.getCollection('books').drop();