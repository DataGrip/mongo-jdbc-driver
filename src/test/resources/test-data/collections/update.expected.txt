|map|
|{0=DeprecationWarning: Collection.update() is deprecated. Use updateOne, updateMany or bulkWrite.}|